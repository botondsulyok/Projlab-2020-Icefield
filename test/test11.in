field stable1 0
field unstable 1 false 0
field stable2 0
neighbours stable1 unstable
neighbours unstable stable1 stable2
neighbours stable2 unstable
entity mover Eskimo stable1
entity puller1 Eskimo stable1
entity puller2 Eskimo stable2
item rope1 Rope puller1
item rope2 Rope puller2
stats stable1
stats unstable
stats stable2
stats mover
stats puller1
stats puller2
move mover unstable
stats mover
stats unstable
stats stable1
stats stable2