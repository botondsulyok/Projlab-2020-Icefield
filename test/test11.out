id: stable1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: mover, puller1
pullers: mover, puller1
masters: mover, puller1
neighbours: unstable

id: unstable
type: unstable
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable1, stable2
capacity: 1
broken: false

id: stable2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: puller2
pullers: puller2
masters: puller2
neighbours: unstable

id: mover
type: eskimo
iceUnit: stable1
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: puller1
type: eskimo
iceUnit: stable1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: rope1
tent: null
components: null

id: puller2
type: eskimo
iceUnit: stable2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: rope2
tent: null
components: null

id: mover
type: eskimo
iceUnit: stable1
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: unstable
type: unstable
building: plains5
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable1, stable2
capacity: 1
broken: true

id: stable1
type: iceunit
building: plains0
snow: 0
pickable: null
standers: puller1, mover
pullers: puller1, mover
masters: puller1, mover
neighbours: unstable

id: stable2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: puller2
pullers: puller2
masters: puller2
neighbours: unstable

